jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/jar/JarVerifier.java

```diff
@@ -146,7 +146,7 @@ public void beginEntry(JarEntry je, ManifestEntryVerifier mev)
                     return;
                 }
 
-                /* J2ObjC modified: copied from sun.misc.JarIndex */
+                /* J2ObjC modified: platform-specific implementation to avoid importing sun.misc.JarIndex */
```

Comment:
I think juts `JarIndex` suffices, since we already commented out importing that class in the beginning of the file.